tests/unit/core/test_config.py

```
@@ -244,3 +247,109 @@ def test_settings_instance(self) -> None:
+        with patch.dict(os.environ, {}, clear=True):
+            settings = AuthSettings()
+            assert settings.ENABLED is False
+            assert settings.BEARER_TOKEN == ""
+
+    def test_custom_values_via_env(self) -> None:
+        """Values are read from AUTH__ prefixed environment variables."""
+        env = {"AUTH__ENABLED": "true", "AUTH__BEARER_TOKEN": "secret"}
+        with patch.dict(os.environ, env, clear=True):
+            settings = AuthSettings()
+            assert settings.ENABLED is True
+            assert settings.BEARER_TOKEN == "secret"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to: "BEARER_TOKEN"

(S105)